src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveterminbarn/ForberedOppgaverTerminbarnService.kt

```diff
@@ -34,15 +34,13 @@ class ForberedOppgaverTerminbarnService(
             val fødselsnummerSøker = fagsakService.hentAktivIdent(terminbarnPåSøknad.first().fagsakId)
             val pdlBarn = pdlBarn(fødselsnummerSøker)
             val ugyldigeTerminbarn = terminbarnPåSøknad.filterNot { it.match(pdlBarn) }
-            lagreOgMapTilOppgaverForUgyldigeTerminbarn(ugyldigeTerminbarn, fødselsnummerSøker, dryRun)
+            lagreOgMapTilOppgaverForUgyldigeTerminbarn(ugyldigeTerminbarn, fødselsnummerSøker)
         }.flatten()
-        logger.info("Fant ${oppgaver.size} oppgaver for ugyldige terminbarn. Dryrun : $dryRun")
+        logger.info("Fant ${oppgaver.size} oppgaver for ugyldige terminbarn.")
         oppgaver.forEach { oppgave ->
-            logger.info("Laget oppgave for behandlingID=${oppgave.behandlingId}. Dryrun : $dryRun")
-        }
-        if (!dryRun) {
-            opprettTaskerForOppgaver(oppgaver)
+            logger.info("Laget oppgave for behandlingID=${oppgave.behandlingId}.")
```

Comment:
Burde vi fjerne denne raden eller endre til eks
```suggestion
            logger.info("Laget oppgave for behandlingID=${oppgave.behandlingId} for ugyldige terminbarn")
```